react/ProductList.tsx

```diff
@@ -47,6 +47,7 @@ const ProductList: FC<Props> = ({ renderAsChildren }) => {
 
   return (
     <div
+      onMouseLeave={e => e.stopPropagation()}
```

Comment:
I'll leave a comment explaining it, I agree this is not clear.

But this is necessary because of the behavior associated with how the `mouseleave` is triggered. If you click on anything inside of the `product-list` that pops over it, such as the quantity-selector component for each product, the `mouseleave` would trigger and bubble up to the event listener placed at `Popup.tsx`, which would close the minicart.

Reply:
You can check if the element being interacted with the mouse is inside the DOM tree. So maybe you don't need this.